src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -3640,36 +3673,6 @@ struct AttentionExecutor : public PagedAttentionExecutor {
             //    W*V aka [m, k1] * [n1, k1]', there is no tails handing for n1, so tails of v_cache need to be set to
             //    zero.
             // for second token, the kernels have tails handling logic
```

Comment:
I think the f32 case which does not touch value-cache in reorder stage may contains invalid values and needs to be zeroed out since the logic here is removed.

Reply:
The padding of value cache is handled in `dequant`

Reply:
I think f32 case may have problem: `q_cache_is_same` will be true, then the `dequant` should not be called:
https://github.com/openvinotoolkit/openvino/blob/c57585a88b840074ef74e3106a8af5d4ba5b3fcd/src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp#L3211-L3219

Reply:
You are right, none-compression kv cache is not well handled.

Reply:
Fix in https://github.com/openvinotoolkit/openvino/pull/30619/commits/5f53f7cc717dae2d9f542822e1f2a0e2ece402d1